packages/driver/cypress/integration/commands/navigation_spec.js

```diff
@@ -825,7 +796,7 @@ describe('src/cy/commands/navigation', () => {
         onLoad,
       })
       .then((win) => {
-        expect(win.bar).to.not.exist
+        expect(win.foo).to.equal('bar')
```

Comment:
why did this test change? I wouldn't have expected this? or was it broken initally?

Reply:
My guess is it wasn't doing anything previously. I'm not sure what it'd be testing for with respect to win.bar in the context of this test.